Xamarin.Essentials/Types/PlatformExtensions/ColorExtensions.shared.cs

```diff
@@ -66,6 +67,124 @@ public static Color GetComplementary(this Color color)
             // multiply by 100 as `ConvertToHsl` specifies them as values between 0 and 1.
             return ColorConverters.FromHsl(h, s * 100, l * 100);
         }
+
+        public static void ToHsv(this Color color, out double h, out double s, out double v)
```

Comment:
I wonder if we can just return a Tuple instead of 3 outs?

Reply:
Maybe something like:

public static (double h, double s, double v) ToHsv(Color color)

Reply:
Besides that we just need docs and good to go :)